instrumentation/opentelemetry-instrumentation-pymongo/src/opentelemetry/instrumentation/pymongo/__init__.py

```diff
@@ -106,7 +108,7 @@ def succeeded(self, event: monitoring.CommandSucceededEvent):
             return
         if span.is_recording():
             span.set_attribute(
-                "db.mongo.duration_micros", event.duration_micros
+                "db.mongodb.duration_micros", event.duration_micros
```

Comment:
looks like this isn't present in semantic conventions. Should we add it?

Reply:
Spec has [call level](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md#call-level-attributes-for-specific-technologies) attributes for specific technologies. It is left to authors of db package to decide whether they want to add it or not. For example one would be interested in number of rows returned by query etc..

Reply:
There are also [language specific attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md#mongodb), and duration_micros is not one of them.

Reply:
Agree. I think the rationale behind including them is that `pymongo` monitoring module provides handful of attributes such as `operation_id`, `request_id`, `duration_micros`, `command_name` and [few](https://api.mongodb.com/python/3.1/api/pymongo/monitoring.html) others and it makes sense to include them with `PymongoInstrumentor` by default.

I am not really sure about sending PR to spec though.

Reply:
@lzchen , I would appreciate your comments on this before opening PR in contrib repo.